corehq/apps/userreports/models.py

```diff
@@ -94,6 +94,7 @@ class DataSourceConfiguration(UnicodeMixIn, CachedCouchDocumentMixin, Document):
     meta = SchemaProperty(DataSourceMeta)
     is_deactivated = BooleanProperty(default=False)
     last_modified = DateTimeProperty()
+    asynchronous = BooleanProperty(default=False)
```

Comment:
I think it'll be a quick way to change a data source that is taking too much time. 

Assuming it'll mostly be static ucrs and likely the eniskhay ones since those have been destroying india's (although much less so since noah's change)